adventureGame/AdventureGame/src/Main.py

```diff
@@ -7,20 +7,37 @@
 from obj import ArmourSet as ArmourSetClass
 from obj import DungeonRoom as DungeonRoomClass
 
+# Make players global
+player1 = None
+player2 = None
+
 def main():
+    # Show the title screen
     titleScreen()
+    
+    # Wait for user to input a key.
     userInput = input()
+    
+    # Clear the terminal screen.
     for i in range(50):
         print("\n")
+    
+    # If the user types "skip", it allows them to skip the introduction story.
     if userInput != "skip":
         introduction()
-    else:    
```

Comment:
Why two instances if "else:" in same loop? One seems to be obsolete...

Reply:
There is only one instance of "else:". Red indicates that it has been removed, green is added. Just a weird git glitch.